### PR TITLE
Chrome extension now prepopulates tenant url even if you don't open the popup menu

### PR DESCRIPTION
**Pull request by andrewckor**

### CODE REVIEW

File: calendar/manifest.json [hunk 1]
```diff
@@ -15,7 +15,10 @@
   },
   "permissions": [
     "storage",
-    "history"
+    "history",
+    "cookies",
+    "*://vowel.com/*",
```
Comment (smatt989) [inline calendar/manifest.json:20]: what is this line of code?

File: calendar/manifest.json [hunk 2]
```diff
@@ -15,7 +15,10 @@
...
+    "*://calendar.google.com/*"
```
Comment (smatt989) [inline calendar/manifest.json:21]: and this one?

File: calendar/popup.js [hunk 3]
```diff
@@ -10,15 +10,15 @@ function click(e) {
...
 function findPreviousTenant() {
-    const r = new RegExp("https:\/\/[a-zA-Z0-9-_]+\.vowel\.com\/\#\/dashboard")
-    chrome.history.search({text: "vowel"}, results => {
-        const mostRecent = results.find(result => r.test(result.url))
-        if(mostRecent) {
-            const tenant = mostRecent.url.split(".")[0].replace("https://","")
-            saveTenant(tenant);
-            getTenant();
-        }
-    })
```
Comment (smatt989) [inline calendar/popup.js:21]: lol RIP this was some very clever code i was happy about hahaha

File: calendar/manifest.json [hunk 4]
```diff
@@ -15,7 +15,10 @@
   },
   "permissions": [
     "storage",
-    "history"
+    "history",
```
Comment (smatt989) [inline calendar/manifest.json:18]: we no longer need history right?

File: calendar/manifest.json (hunk 1 above)
Comment (andrewckor) [reply calendar/manifest.json:20]: So we first get access to cookies API, like we did with history and secondly we define which domains we get access to read cookies and send message events. That means we can read only cookies that are set for vowel.com, and nothing else.

File: calendar/manifest.json (hunk 2 above)
Comment (andrewckor) [reply calendar/manifest.json:21]: This is to make sure we get access to send messages from the content script to the background script only on these whitelisted domains.

File: calendar/popup.js (hunk 3 above)
Comment (andrewckor) [reply calendar/popup.js:21]: It was great actually.

File: calendar/manifest.json (hunk 4 above)
Comment (andrewckor) [reply calendar/manifest.json:18]: Correct.

Comment (andrewckor) [reply calendar/manifest.json:18]: I will remove it.

**Pull request merged**